indico/modules/rb_new/client/js/common/bookings/BookingEdit.jsx

```diff
@@ -0,0 +1,622 @@
+                           position="right center"
+                           content={<TimelineLegend labels={legendLabels} compact />} />
+                    <Label.Group styleName="occurrence-count">
+                        <Popup trigger={<Label color="blue" size="tiny" content={this.numberOfOccurrences} circular />}
+                               position="bottom center"
+                               on="hover">
+                            <Translate>
+                                Number of occurrences for this booking
+                            </Translate>
+                        </Popup>
+                        {numberOfNewCandidates > 0 && (
+                            <>
+                                {' → '}
+                                <Popup trigger={<Label color="grey" size="tiny" content={numberOfNewCandidates}
+                                                       circular />}
+                                       position="bottom center"
+                                       on="hover">
+                                    <Translate>
+                                        Number of occurrences for this booking after changes
+                                    </Translate>
+                                </Popup>
+                            </>
+                        )}
+                    </Label.Group>
+                </Header>
+                <DailyTimelineContent rows={isFetching ? [] : dateRange.map(serializeRow)}
+                                      fixedHeight={dateRange.length > 1 ? '100%' : null}
+                                      isLoading={isFetching} />
+            </div>
+        );
+    };
+
+    renderConflictsMessage = () => {
+        const {skipConflicts} = this.state;
+
+        if (this.numberOfConflicts > 0) {
+            return (
+                <div styleName="booking-conflicts-info">
+                    <Message color="yellow" attached icon>
+                        <Icon name="warning circle" />
+                        <Message.Content>
+                            <Message.Header>
+                                <Translate>Conflicts with new booking</Translate>
+                            </Message.Header>
+                            <PluralTranslate count={this.numberOfConflicts}>
+                                <Singular>
+                                    Your new booking might conflict with another one.
+                                </Singular>
+                                <Plural>
+                                    <Param name="count" value={this.numberOfConflicts} /> occurrences of your
+                                    new booking might conflict with existing bookings.
+                                </Plural>
+                            </PluralTranslate>
+                        </Message.Content>
+                    </Message>
+                    {this.numberOfCandidates > this.numberOfConflicts && (
```

Comment:
why is checking `this.numberOfConflicts > 0` not enough?

Reply:
Since you can have e.g. one conflicts for a single day booking which would result in a booking with zero valid occurrences. So there is no point in letting a user skip "conflicting" occurrences if `numOfOccurrences === numOfConflicts`